perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/setup_host.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Installing ops agent
+curl -sSO https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh 
+sudo bash add-google-cloud-ops-agent-repo.sh --also-install --version=latest
+
+# Installing docker engine
+echo "Installing linux utility packages, like, lsb-release, curl..."
+sudo apt-get update
+sudo apt-get install -y \
+    ca-certificates \
+    curl \
+    gnupg \
+    lsb-release
+
+echo "Installing docker framework..."
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+  $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+sudo apt-get update
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-compose-plugin -y
+
+echo "Installing driver..."
+sudo apt update && sudo apt install -y build-essential
+BASE_URL=https://us.download.nvidia.com/tesla
+DRIVER_VERSION=450.172.01
+sudo curl -fSsl -O $BASE_URL/$DRIVER_VERSION/NVIDIA-Linux-x86_64-$DRIVER_VERSION.run
+
+sudo sh NVIDIA-Linux-x86_64-$DRIVER_VERSION.run -s
+
+echo "Installing NVIDIA container tool..."
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID) \
+      && curl -fsSL https://nvidia.github.io/libnvidia-container/gpgkey | sudo gpg --dearmor -o /usr/share/keyrings/nvidia-container-toolkit-keyring.gpg \
+      && curl -s -L https://nvidia.github.io/libnvidia-container/$distribution/libnvidia-container.list | \
+            sed 's#deb https://#deb [signed-by=/usr/share/keyrings/nvidia-container-toolkit-keyring.gpg] https://#g' | \
+            sudo tee /etc/apt/sources.list.d/nvidia-container-toolkit.list
+
+sudo apt-get update
+sudo apt-get install -y nvidia-docker2
+sudo systemctl restart docker
+
+echo "Copying tf util files from bucket"
+gsutil -m cp gs://abhinav-working-dir/resnet.py ./perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/
+gsutil -m cp gs://abhinav-working-dir/files-modified-2.10.0/controller.py ./perfmetrics/scripts/continuous_test/ml_tests/tf/setup_scripts/
```

Comment:
did you try running this? I see these copy commands are run from [this code line](https://github.com/GoogleCloudPlatform/gcsfuse/blob/f4db1eb48354cc2c8fe5ba7db26273292d18e264/perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh#L7) and current working directory at that line is [not gcsfuse/](https://github.com/GoogleCloudPlatform/gcsfuse/blob/f4db1eb48354cc2c8fe5ba7db26273292d18e264/perfmetrics/scripts/continuous_test/ml_tests/tf/build.sh#L3).